test/orders-test.js

```diff
@@ -82,7 +82,7 @@ suite('get orders', function() {
     .end(function(err, res) {
       if (err) return done(err);
 
-      assert.strictEqual(res.body.orders.length, 34);
+      assert.strictEqual(res.body.orders.length, 36);
```

Comment:
Pretty sure it's because of these lines

```
{
          "flags": 0,
          "seq": 844069,
          "taker_gets": {
            "currency": "USD",
            "issuer": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B",
            "value": "40"
          },
          "taker_pays": "1000000"
        }
```